db/seeds.rb

```diff
@@ -56,21 +57,61 @@
   end
 end
 
-# Add a note to Patient 2
+# Patients should have names that reflect what we're looking for (e.g. 'Patient With Other Contact' rather than 'Patient 2')
+# Add example of patient with other contact info
+  patient1 = Patient.find_by( name: "Patient 1")
```

Comment:
This is dope stuff -- going to be SUPER useful for reviewing test cases. 

What are your thoughts about moving some of this closer to line 22, on actual point of patient creation? Worth the trouble, or probably nobody will ever really dig into this so whatever?